packages/docusaurus-theme-classic/src/theme-classic.d.ts

```diff
@@ -1510,3 +1510,7 @@ declare module '@theme/prism-include-languages' {
     PrismObject: typeof PrismNamespace,
   ): void;
 }
+
+declare module '@theme/DocBreadcrumbs/HomeBreadcrumbItem' {
```

Comment:
Considering we are going to have multiple items I'd prefer to have something like `@theme/DocBreadcrumbs/Items/Home`